### PR TITLE
Incompany requests

### PR DESCRIPTION
**Pull request by V-Ho**
Index of Incompany Requests
screenshot from 2017-03-31 12-33-07

### CODE REVIEW

**Comment by V-Ho** [comment]
Edit Form
screenshot from 2017-03-31 12-33-36

**Comment by V-Ho** [comment]
Delete form
screenshot from 2017-03-31 12-34-06

File: app/assets/javascripts/incompany_requests.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```
Comment (Woulei) [inline app/assets/javascripts/incompany_requests.coffee]: Don't use it? => Delete it!

File: app/assets/stylesheets/incompany_requests.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the IncompanyRequests controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
```
Comment (Woulei) [inline app/assets/stylesheets/incompany_requests.scss]: Don't use it? => Delete it!

File: app/helpers/incompany_requests_helper.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+module IncompanyRequestsHelper
+end
```
Comment (Woulei) [inline app/helpers/incompany_requests_helper.rb]: Don't use it? => Delete it!

**Pull request merged**